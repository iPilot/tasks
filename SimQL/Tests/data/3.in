{
    "data": {
        "empty": [],
        "x": [ 0.1, 0.2, 0.3 ],
        "a": [
            {
                "b": 10,
                "c": [ 1, 2, 3 ]
            },
            {
                "b": 30,
                "c": [ 4 ]
            },
            { "d": 500 }
        ]
    },
    "queries": [ "sum(empty)", "sum(a.b)", "sum(a.c)", "sum(a.d)", "sum(x)" ]
}