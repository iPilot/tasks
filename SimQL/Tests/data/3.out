sum(empty) = 0
sum(a.b) = 40
sum(a.c) = 10
sum(a.d) = 500
sum(x) = 0.6